### ISSUE TITLE
Engine -> Framework roll blocked on customer_testing failure

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#100906]
Starting with https://github.com/flutter/flutter/pull/100901, which contains only the Dart roll https://github.com/flutter/engine/pull/32291. customer_tests is failing with

```
Error: Setter not found: 'AsciiChar'.
|   set AsciiChar(int value) => Char.AsciiChar = value;
|                                    ^^^^^^^^^
```

From the rainbowmonkey app: https://cirrus-ci.com/task/6028198742851584.

It looks like https://dart.googlesource.com/sdk.git/+/90f118484f78fe2588c796969b2f35aef88d987e is the culprit.

cc @dcharkes @mkustermann @a-siva

**Comment by a-siva** [comment]
This appears to be a duplicate of https://github.com/timsneath/win32/issues/349
Do we need to use the latest version of win32 package to resolve this ?

**Comment by dnfield** [comment]
I believe RainbowMonkey ends up having issues with this because it both uses `any` constraints and checks in its pubspec.lock, which means it kind of gets the worst of all possible worlds (tight version constraints based on whenever the last time pub get was run and checked in)....

**Comment by Hixie** [comment]
Should be easy to fix, just run `dart pub upgrade --major-vesions` to get the new dependencies.

**Comment by zanderso** [comment]
> Should be easy to fix, just run `dart pub upgrade --major-vesions` to get the new dependencies.

https://github.com/jocosocial/rainbowmonkey/pull/344

**Comment by zanderso** [comment]
We deleted rainbowmonkey from customer_testing: https://github.com/flutter/tests/commit/69c3406c7f26b5238bff7b0777d2521cf0c74de0. I will restart the roller.

**Comment by zanderso** [comment]
Rolls are going again.

**Comment by gaozm0509** [comment]
run ```flutter pub upgrade --major-versions``` can solve it
